packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -212,7 +212,7 @@ contract L2Genesis is Deployer {
     ///         migrating genesis generation to Solidity.
     function setPredeployProxies() public {
         console.log("Setting Predeploy proxies");
-        bytes memory code = vm.getDeployedCode("Proxy.sol:Proxy");
+        bytes memory code = vm.getDeployedCode("forge-artifacts/Proxy.sol/Proxy.json");
```

Comment:
just curios what necessitated this?

Reply:
0Zs proxy situated at forge-artifacts/proxy/Proxy.sol/Proxy.json means there are 2 Proxy.sol's. Basically 2 artifacts with the same name

Reply:
I see, but I'm confused why that only an issue now? The only affected submodule I see is packages/contracts-bedrock/lib/lib-keccak

Reply:
I think that's because both are now being compiled with the same solc version. still uncertain how that made this an issue now

Reply:
Do we intend to use OZ's own or ours?

Reply:
Definitely our own.

Reply:
> 0Zs proxy situated at forge-artifacts/Proxy.sol/Proxy.json means there are 2 Proxy.sol's. Basically 2 artifacts with the same name

So... It sounds like you're saying that as-is, we're using OZ's Proxy.sol?

Reply:
Yeah that's why I asked, I'm going to change that now

Reply:
It looks like OZ Proxy.sol is [`abstract`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.3/contracts/proxy/Proxy.sol#L16), so I don't think we are using it.

Reply:
Looks like theirs is in `forge-artifacts/proxy/Proxy.sol/Proxy.json`

Reply:
ah yes that's truee, i can confirm that

Reply:
No need to do that as part of this PR ^